README.md

```diff
@@ -225,10 +225,11 @@ GET /status                              - returns middleware status
 
 ## OpenAPI specs
 
-Under the `/swagger` path there are two files providing a swagger specificiation of the endpoints for both api versions:
+There are two files providing a swagger specificiation of the endpoints for respective api versions:
+
+- https://mainnet.aeternity.io/mdw/swagger/swagger_v1.yaml
```

Comment:
can we add the links so they open in a hosted (e.g. petstore) swagger ui?

Reply:
That was my first comment on the PR (: swagger editor imports the file, will try replacing the pet store.

Reply:
> can we add the links so they open in a hosted (e.g. petstore) swagger ui?

The guys from swagger UI had thought about it (: It requires a form request controlled by javascript. The same for other examples I have found.

Reply:
> can we add the links so they open in a hosted (e.g. petstore) swagger ui?

The guys from swagger UI had thought about it (: It requires a form request controlled by javascript. The same for other examples I have found.

Reply:
you can add `?url=` to the domain

Reply:
was falling on the validator, thanks!